tests/graphql_fragments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fragments.graphql 1`] = `
+{
+  field
+  ...   XYZ
+  ... on TheType {
+    ...AFragment
+    ...          {
+      noTypeCondition
+    }
+  }
+}
+
+fragment XYZ on ABC {
+  field
+  ...AFragment
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{
+  field
+  ...XYZ
+  ... on TheType {
```

Comment:
Yeah this is the preferred style according to @leebyron as far as I know. The idea is that it's like the spce after the function keyword in a function expression in JS I think.

I do think it might be better to always have a space there, even for named fragments.